src/emscripten-optimizer/parser.h

```diff
@@ -186,988 +184,6 @@ extern std::vector<OperatorClass> operatorClasses;
 extern bool isIdentInit(char x);
 extern bool isIdentPart(char x);
 
-// parser
```

Comment:
This parses JS, which I guess could be a separate PR. But maybe not worth it if it's just a safe removal.